test/Interop/Cxx/class/inheritance/virtual-methods-typechecker.swift

```diff
@@ -2,4 +2,13 @@
 
 import VirtualMethods
 
+let _ = Base() // expected-error {{'Base' cannot be constructed because it has no accessible initializers}}
```

Comment:
can we give a better diagnostic here, that reference the fact that the C++ class is abstract?

Reply:
Good point, let's mark the constructor as unavailable in Swift